DiscordChatExporter.Core/Discord/Data/ChannelThread.cs

```diff
@@ -11,17 +11,27 @@ public record ChannelThread(
     ChannelKind Kind,
     Snowflake GuildId,
     Snowflake ParentId,
+    string? ParentName,
+    int? ParentPosition,
     string Name,
+    int? Position,
+    string? IconUrl,
+    string? Topic,
```

Comment:
You don't have to put these unused parameters in the constructor, just add a separate properties inside this record:

```csharp
public int? Position => null;
public string? IconUrl => null;
public stirng? Topic => null;
```

etc

Reply:
Got it. I'd assume you mean something along the lines of what I've just done in 9ed1238?